fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/translation/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * package-info.java
```

Comment:
Should we move `TranslationMap` into this package? We can do it as a last step as that move will probably cause a ton of diffs.

Reply:
That makes sense to me, let me do that and see what happens.